securesystemslib/signer/_hsm_signer.py

```diff
@@ -192,15 +228,56 @@ def _find_key_values(
         return ECDomainParameters.load(bytes(params)), bytes(point)
 
     @classmethod
-    def import_(cls, hsm_keyid: Optional[int] = None) -> Tuple[str, SSlibKey]:
+    def _build_token_filter(cls) -> Dict[str, str]:
+        """Builds a token filter for the found slot/token.
+
+        The filter will include 'label' if one is found on token.
+
+        raises ValueError if less or more than 1 token/slot is found
```

Comment:
Is it worth disambiguating the terms "slot" and "token" somewhere in the HSM signer documentation? I must admit, I'm still confused. Also because they mean different things in related standards (pkcs11 and piv).

Reply:
I guess I could remove any mention of slots. 

The slot I mean here is the PKCS11 slot, aka the smart card reader: since we don't filter readers I don't think we even need to mention them